frontend/javascripts/dashboard/advanced_dataset/create_explorative_modal.js

```diff
@@ -0,0 +1,149 @@
+        checked={isFallbackSegmentationSelected}
+        disabled={!isFallbackSegmentationOptional}
+      >
+        With Existing Segmentation{" "}
+        <Tooltip
+          title="Base your volume annotation on an existing segmentation layer of this dataset. Note that skeleton-only annotations always show the existing segmentation."
+          placement="right"
+        >
+          <InfoCircleOutlined />
+        </Tooltip>
+      </Checkbox>
+    );
+
+    const lowResolutionIndex = Math.max(lowestResolutionIndex, userDefinedResolutionIndices[0]);
+    const highResolutionIndex = Math.min(highestResolutionIndex, userDefinedResolutionIndices[1]);
+
+    modalContent = (
+      <React.Fragment>
+        <Radio.Group onChange={e => setAnnotationType(e.target.value)} value={annotationType}>
+          <Radio value="hybrid">Skeleton and Volume</Radio>
+          <Radio value="skeleton">Skeleton only</Radio>
+          <Radio value="volume">Volume only</Radio>
+        </Radio.Group>
+        <br />
+        <br />
+        {fallbackCheckbox}
+        <br />
+        <br />
+        <h5>
+          Volume Resolutions{" "}
+          <Tooltip
+            title="Select which of the dataset resolutions the volume data should be created at. Resolution 1 is the most detailed, 4-4-2 is downsampled by factor 4 in x and y, and by factor 2 in z."
+            placement="right"
+          >
+            <InfoCircleOutlined />
+          </Tooltip>
+        </h5>
+        <div style={{ display: "inline-block", width: "5em", verticalAlign: "middle" }}>
+          {datasetResolutionInfo.getResolutionByIndexOrThrow(lowResolutionIndex).join("-")}
+        </div>
+        <div
+          style={{
+            display: "inline-block",
+            width: "60%",
+            verticalAlign: "middle",
+            paddingRight: "1em",
+          }}
+        >
+          <Slider
+            tooltipVisible={false}
+            onChange={value => setUserDefinedResolutionIndices(value)}
+            range
+            disabled={annotationType === "skeleton"}
+            step={1}
+            min={lowestResolutionIndex}
+            max={highestResolutionIndex}
+            value={[lowResolutionIndex, highResolutionIndex]}
+          />
+        </div>
+        <div style={{ display: "inline-block", width: "5em", verticalAlign: "middle" }}>
+          {datasetResolutionInfo.getResolutionByIndexOrThrow(highResolutionIndex).join("-")}
+        </div>
+        <br />
+        <br />
+        <Link
+          to={`/datasets/${dataset.owningOrganization}/${
+            dataset.name
+          }/createExplorative/${annotationType}/${isFallbackSegmentationSelectedString}?minRes=${Math.max(
+            ...datasetResolutionInfo.getResolutionByIndexOrThrow(lowResolutionIndex),
+          )}&maxRes=${Math.max(
+            ...datasetResolutionInfo.getResolutionByIndexOrThrow(highResolutionIndex),
+          )}`}
```

Comment:
I'm wondering whether there can be a scenario where minRes/maxRes should not be passed? I cannot think of any realistic scenarios, though. Maybe you can give it a thought, too, if you haven't already.

Reply:
I think it’s fine to pass it always. Selecting the full range will yield the same end result as passing nothing.